scripts/utilities/update_revision.sh

```diff
@@ -78,13 +84,30 @@ source ${1};
 
 # If ATSRC_PACKAGE_CO is not defined, we skip the revision check.
 if ! [ -n "${ATSRC_PACKAGE_CO}" ] || ! [ -n "${ATSRC_PACKAGE_REV}" ]; then
-	return 0;
+	exit 0;
 fi
 
-# This function print a message with identation
+if [[ ${service} = "github" ]];
+then
+	if [[ "${5}" = "" ]];
+	then
+		GIT_USER=$(git config --global user.name)
+		GIT_EMAIL=$(git config --global user.email)
+
+		if [[ "${GIT_USER}" = "" ]] || [[ "${GIT_EMAIL}" = "" ]];
+		then
+			echo "Error: Git's user.name or user.email not configured."
+			exit 1;
+		fi
+
+		GITHUB_SIGNATURE="${GIT_USER} <${GIT_EMAIL}>"
```

Comment:
It won't because it checks if the 5th argument is "" before assigning a new value to $GITHUB_SIGNATURE. But I'll replace the "${5}" = "" check on line 92 with "$GITHUB_SIGNATURE" = "" to make it more intuitive :)